resources/scripts/jquery/jquery-utils.js

```diff
@@ -308,6 +308,30 @@
     	$(".eXist_wizard_next", container).click(function () {
     		if (currentPage == pages.length - 1)
     			return;
+		var name = document.getElementsByName('name')[0].value;
+           	var abbrev = document.getElementsByName('abbrev')[0].value;
+           	var title = document.getElementsByName('title')[0].value;
+
+                if (name.startsWith('urn:') || name.startsWith('file:')) {
+                    alert("The name cannot be in a urn:// or a file:// protocol.")
+                    return false;} else {
+                       try {
+                           nameURL = new URL(name);
+                           } catch (_) {
+                           return alert("Enter a valid URI as name.");
```

Comment:
In all the other bad cases false is returned. Would it make sense to do it here as well?

Reply:
The `catch` already prevents the default action. At least that's what it does in my test.